examples/all-clusters-app/linux/AllClustersCommandDelegate.cpp

```diff
@@ -769,6 +803,71 @@ void AllClustersAppCommandHandler::OnAirQualityChange(uint32_t aNewValue)
     }
 }
 
+void AllClustersAppCommandHandler::HandleSetOccupancyChange(EndpointId endpointId, uint8_t newOccupancyValue)
+{
+    BitMask<chip::app::Clusters::OccupancySensing::OccupancyBitmap> currentOccupancy;
+    Protocols::InteractionModel::Status status = OccupancySensing::Attributes::Occupancy::Get(endpointId, &currentOccupancy);
+
+    if (static_cast<BitMask<chip::app::Clusters::OccupancySensing::OccupancyBitmap>>(newOccupancyValue) == currentOccupancy)
+    {
+        ChipLogDetail(NotSpecified, "Skipping setting occupancy changed due to same value.");
+        return;
+    }
+
+    status = OccupancySensing::Attributes::Occupancy::Set(endpointId, newOccupancyValue);
+    ChipLogDetail(NotSpecified, "Set Occupancy attribute to %u", newOccupancyValue);
+
+    if (status != Protocols::InteractionModel::Status::Success)
+    {
+        ChipLogDetail(NotSpecified, "Invalid value/endpoint to set.");
+        return;
+    }
+
+    EmitOccupancyChangedEvent(endpointId, newOccupancyValue);
+
+    if (1 == newOccupancyValue)
```

Comment:
Using the enum value would be easier to follow

Reply:
@jmartinez-silabs Would this comment require some follow up action?  Otherwise, may I close the conversation?

Reply:
Will follow-up